src/h3lib/lib/geoCoord.c

```diff
@@ -221,36 +221,36 @@ void _geoAzDistanceRads(const GeoCoord* p1, double az, double distance,
         if (fabs(p2->lat - M_PI_2) < EPSILON)  // north pole
         {
             p2->lat = M_PI_2;
-            p2->lon = 0.0L;
+            p2->lon = 0.0;
```

Comment:
What if anything is the affect of removing the `L` suffix? Do we lose precision? I'm guessing no, just wanted to understand if this is expected to be a no-op.

Reply:
`p2->lon` is a double, so this should have the same result.

The comparisons using long doubles are different, I assume in those cases the double variables are promoted to long doubles for the purpose of comparison. I believe that could possibly have issues in edge cases.